namer/dnssrv/src/main/scala/io/buoyant/namer/dnssrv/DnsSrvNamer.scala

```diff
@@ -12,19 +13,26 @@ import scala.util.control.NoStackTrace
 
 class DnsSrvNamer(
   prefix: Path,
-  resolver: DNS.Resolver,
+  resolver: () => DNS.Resolver,
   refreshInterval: Duration,
-  stats: StatsReceiver,
-  pool: FuturePool
+  stats: StatsReceiver
 )(implicit val timer: Timer)
   extends Namer {
 
   override def lookup(path: Path): Activity[NameTree[Name]] = memoizedLookup(path)
 
   private[this] val success = stats.counter("lookup_successes_total")
   private[this] val failure = stats.counter("lookup_failures_total")
+  private[this] val dnsscope = stats.scope(s"lookup_dns")
```

Comment:
I don't think you need these to be interpolation (s) strings

Reply:
fixed